addons/payment_demo/__manifest__.py

```diff
@@ -12,6 +12,7 @@
         'views/payment_demo_templates.xml',
         'views/payment_token_views.xml',
         'views/payment_transaction_views.xml',
+        'views/payment_provider_views.xml',
```

Comment:
Alphabetical order, unless adding this line here is required by data dependency, but then add a comment.